Client/Cliqz/Frontend/Resources/fr.lproj/Cliqz.strings

```diff
@@ -1,144 +1,144 @@
-"Fair Blocking" = "Fair Blocking";
+"Fair Blocking" = "Bloquage équitable";
 
 /* More details about Fair Blocking */
-"What is 'fair'?" = "What is 'fair'?";
+"What is 'fair'?" = "Qu'est-ce que 'équitable'?";
 
 /* Settings item for clearing favorite history */
-"Favorites" = "Favorites";
+"Favorites" = "Favoris";
 
 /* Settings item for clearing private data on termination */
-"Clear Private data on termination" = "Clear Private Data on Termination";
+"Clear Private data on termination" = "Supprimer les données privées à la fermerture";
 
 /* Enable News Push Notifications */
-"Enable News Push Notifications" = "News Notification";
+"Enable News Push Notifications" = "News Notification"; # context? does 'Alert for News'
 
 /* Show blocked top-sites from settings */
-"Show blocked topsites" = "Restore Most Visited Websites";
+"Show blocked topsites" = "Restaurer les sites les plus visités";
 
 /* Text of the 'Show blocked top-sites' alert */
-"All blocked topsites will be shown on the start page again." = "Start page shows most-visited websites again.";
+"All blocked topsites will be shown on the start page again." = "La page d'accueil montre les sites les plus visités.";
 
 /* OK button in the 'Show blocked top-sites' alert */
 /* Button label to `OK` in news notification permission screen */
 "OK" = "OK";
 
 /* Cancel button in the 'Show blocked top-sites' alert */
-"Cancel" = "Cancel";
+"Cancel" = "Annuler";
 
 /* Button label to `Yes please!` in news notification permission screen */
-"Yes please!" = "Yes please!";
+"Yes please!" = "Oui SVP!";
 
 /* Button label to `Later` in news notification permission screen */
-"Remind me later" = "Remind me later";
+"Remind me later" = "Me rappeler plus tard";
 
 /* Button label to `No` in news notification permission screen */
-"No thanks" = "No thanks";
+"No thanks" = "Non merci";
 
 /* Description text displayed when there are no open tabs while in Forget mode */
-"Forget Mode" = "Forget Mode";
+"Forget Mode" = "Mode oubli";
 
 /* Description text displayed when there are no open tabs while in Forget mode */
-"Browsing history and cookies will not be remembered, but new downloads and bookmarks will be saved." = "Browsing history and cookies will not be remembered, but new downloads and bookmarks will be saved.";
+"Browsing history and cookies will not be remembered, but new downloads and bookmarks will be saved." = "L'historique de navigation ainsi que les cookies seront supprimés. Toutefois, les téléchargements et le marque-pages seront sauvegardés.";
 
 /* Context/Sharing menu item for downloading youtube videos */
-"Download youtube video" = "view video offline";
+"Download youtube video" = "Regarder la vidéo en mode hors connexion";
 
 /* History title on dashboard */
-"History" = "History";
+"History" = "Historique";
 
 /* Add Favorites */
-"Add Favorite" = "Add Favorite";
+"Add Favorite" = "Ajouter aux Favors";
 
 /* Setting menu title */
-"Settings" = "Settings";
+"Settings" = "Paramètres";
```

Comment:
I just checked on Firefox. they are using: 
- menu
- préférences
i would suggest `menu`, because there is not that many space on the Browser.